general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -546,6 +546,67 @@ def process_aces(self, job_id: str) -> list[float]:
         """
         return self.post_request("/aces_fetch", {"job_id": job_id})
 
+    def submit_cb(
+        self,
+        target: str,
+        shots: int,
+        serialized_circuits: dict[str, str],
+        num_channels: int,
+        num_sequences: int,
+        depths: list[int],
+        method: str | None = None,
+        noise: dict[str, object] | None = None,
+    ) -> str:
+        """Makes a POST request to the `/cycle_benchmarking` endpoint.
+
+        Args:
+            target: The device target to characterize.
+            shots: How many shots to use per circuit submitted.
+            serialized_circuits: The serialized process circuits to use in the protocol.
+            num_channels: The number of random Pauli decay channels to approximate error.
+            num_sequences: Number of circuits to generate per depth.
+            depths: Lists of depths representing the depths of Cycle Benchmarking
+                circuits to generate.
+            method: Optional method to use in device submission (e.g. "dry-run").
+            noise: Dictionary representing noise model to simulate the protocol with.
+
+        Returns:
+            A string with the job id for the ACES job created.
+
+        Raises:
+            ValueError: If the target or noise model is not valid.
+            SuperstaqServerException: If the request fails.
+        """
+        gss.validation.validate_target(target)
+
+        json_dict: dict[str, Any] = {
+            **serialized_circuits,
+            "target": target,
+            "shots": shots,
+            "num_channels": num_channels,
+            "num_sequences": num_sequences,
+            "depths": depths,
+        }
+
+        if method:
+            json_dict["method"] = method
+        if noise:
+            json_dict["noise"] = noise
+
+        return self.post_request("/cycle_benchmarking", json_dict)
```

Comment:
definitely not super important but if feel like it's a little inconsistent to use `"/cycle_benchmarking"` and `"/cb_fetch"` - maybe this should be `"/cb_submit"`?